compiler/util/clangUtil.cpp

```diff
@@ -54,6 +54,8 @@
 
 #include "llvmDebug.h"
 
+#include "llvm/Support/TargetRegistry.h"
```

Comment:
Oh well, apparently I forgot to remove this include, it's not necessary for anything to work.